content/en/guides/models/app/features/panels/line-plot/_index.md

```diff
@@ -168,4 +169,52 @@ From a line plot, you can quickly create a [run metrics notification]({{< relref
+
+### Find infrastructure issues in a run
+
+W&B surfaces both SLURM job issues and cluster node issues. View infrastructure errors in a run:
+
+1. Navigate to your project on the W&B App. 
+2. Select the **Workspace** tab to view your project’s workspace.
+3. Search and select the name of the run that contains an infrastructure issue. One or more red vertical lines with an exclamation mark overlay plots indicating an infrastructure issue. 
+4. Select an issue on a plot or select the **Issues** button in the top right of the page. A modal appears that lists each issue reported by CoreWeave. 
+
+{{< alert title="Tip" >}}
+Pin the **Issues** column to your W&B Workspace to view runs that logged an issue at a glance. For more information about how to pin a column, see [Customize how runs are displayed]({{< relref "/guides/models/track/runs/#customize-how-runs-are-displayed" >}}).
+{{< /alert >}}
+
+The **Overall Grafana view** at the top of the modal redirects you to the SLURM job's Grafana dashboard, which contains system-level details about the run. The **Issue** summary describes the root error that the SLURM job reported to CoreWeave Mission Control. The summary section also describes any attempts to automatically resolve the error made by CoreWeave.
+
+The **All Issues** list all issues that occurs during the run in chronological order, with the most recent issue at the top. The list contains the job issue and node issue alerts. Within each issue alert is the name of the issue, the timestamp when the issue occurred, a link to the Grafana dashboard for that issue, and a brief summary that describes the issue.
+
+The following table lists the categories of infrastructure issues and example alerts:
+
+| Category | Example alerts |
+| -------- | ------------- |
+| Node Availability & Readiness | `KubeNodeNotReadyHGX`, `NodeExtendedDownTime` |
+| GPU/Accelerator Errors | `GPUFallenOffBusHGX`, `GPUFaultHGX`, `NodeTooFewGPUs` |
+| Hardware Errors | `HardwareErrorFatal`, `NodeRAIDMemberDegraded` |
+| Networking & DNS | `NodeDNSFailureHGX`, `NodeEthFlappingLegacyNonGPU` |
+| Power, Cooling, and Management | `NodeCPUHZThrottle`, `RedfishDown` |
+| DPU & NVSwitch | `DPUNcoreVersionBelowDesired`, `NVSwitchFaultHGX` |
+| Miscellaneous | `NodePCISpeedRootGBT`, `NodePCIWidthRootSMC` |
+
+For detailed information on error types, see the [SLURM Job Metrics on the CoreWeave Docs](https://docs.coreweave.com/docs/observability/managed-grafana/slurm-job-metrics#job-info-alerts).
+
+### Debug infrastructure issues
+Each run that you create in W&B corresponds to a single SLURM job in CoreWeave. You can view a failed job’s Grafana dashboard or discover more information about a single node. The link within the **Overview** section of the Issues modal links to the SLURM job Grafana dashboard. 
```

Comment:
I'm having trouble understanding this section. To check my understanding, does it mean (not a suggested edit) "If a run had infrastructure issues, you can learn more on the Grafana dashboard. From there, you can view details about each failed Slurm job and the node that ran it"?